.clang-tidy

```diff
@@ -1,10 +1,8 @@
 ---
-#Checks:          '*,-clang-diagnostic-*'
-Checks:          '-clang-*,performance*'
-#WarningsAsErrors: '*'
-#HeaderFilterRegex: '\/src\/'
+Checks:          'clang-*,performance*,-performance-move-const-arg'
```

Comment:
I disabled the `performance-move-const-arg` check here due to discussed false positives.